go/api/handlers/user.go

```diff
@@ -16,6 +16,19 @@ type UserHandler struct {
 	Repository repository.UserRepository
 }
 
+type UserResponse struct {
+	ID       uint          `json:"id"`
+	Username string        `json:"username"`
+	Trips    []models.Trip `json:"trips"`
+}
+
+type UserAdminResponse struct {
+	ID       uint          `json:"id"`
+	Username string        `json:"username"`
+	Trips    []models.Trip `json:"trips"`
```

Comment:
```suggestion
	UserResponse
```
Hérite de user response